### PR TITLE
RDMA: Enable multiple port testing

### PR DESCRIPTION
**Pull request by RoiZilberzwaig**

### CODE REVIEW

File: ngc_rdma_test.sh [hunk 1]
```diff
@@ -103,28 +107,53 @@ $0 client mlx5_0 server mlx5_3 --conn=UC,UD,DC
 EOF
 }
 
+# Checks if the number of command-line arguments ($#) is not equal to 4.
+# If so, the script calls to show_help function to display help or usage information.
```
Comment (blochl) [inline ngc_rdma_test.sh:111]: The above is kinda self-evident, no? :) ([Self-documenting code](https://swimm.io/learn/documentation-tools/tips-for-creating-self-documenting-code) is awesome)

File: ngc_rdma_test.sh [hunk 2]
```diff
@@ -103,28 +107,53 @@ $0 client mlx5_0 server mlx5_3 --conn=UC,UD,DC
...
+    read -ra server_QPS <<< $(default_qps_optimization "$SERVER_TRUSTED" "${SERVER_DEVICES[@]}")
+fi
 #Defaults are not using cuda, set params as empty string
 server_cuda=""
 client_cuda=""
 server_cuda2=""
 client_cuda2=""
 
-if (( $# != 4  ))
+BASE_TCP_POTR=10000
```
Comment (blochl) [inline ngc_rdma_test.sh:139]: s/POTR/PORT/

File: ngc_rdma_test.sh [hunk 3]
```diff
@@ -103,28 +107,53 @@ $0 client mlx5_0 server mlx5_3 --conn=UC,UD,DC
...
+
+if [ "${#SERVER_DEVICES[@]}" -ne "${#CLIENT_DEVICES[@]}" ]
 then
-    show_help
-    exit 1
+        fatal "The number of server and client devices must be equal."
```
Comment (blochl) [inline ngc_rdma_test.sh:143]: Indentation.

File: ngc_rdma_test.sh [hunk 4]
```diff
@@ -340,4 +236,4 @@ for TEST in "${TESTS[@]}"; do
             fi
         done
     done
-done
+done
```
Comment (blochl) [inline ngc_rdma_test.sh:239]: Please re-add the newline.

**Review by blochl — CHANGES_REQUESTED** [review]

File: ngc_rdma_test.sh (hunk 1 above)
Comment (RoiZilberzwaig) [reply ngc_rdma_test.sh]: Fixed!

File: ngc_rdma_test.sh (hunk 2 above)
Comment (RoiZilberzwaig) [reply ngc_rdma_test.sh]: Fixed!

File: ngc_rdma_test.sh (hunk 3 above)
Comment (RoiZilberzwaig) [reply ngc_rdma_test.sh]: Fixed!

File: ngc_rdma_test.sh (hunk 4 above)
Comment (RoiZilberzwaig) [reply ngc_rdma_test.sh]: Fixed!

**Comment by RoiZilberzwaig** [comment]
Thanks Leonid!
All comments you voted on have been fixed.

**Review by blochl — APPROVED** [review]
Thanks, @RoiZilberzwaig! Awesome work!

**Pull request merged**